src/chain_spec.rs

```diff
@@ -115,6 +119,22 @@ fn polymath_props() -> Properties {
         .clone()
 }
 
+fn currency_codes() -> Vec<Ticker> {
+    let currency_file = fs::read_to_string("src/data/currency_symbols.json").unwrap();
+    let currency_data: FiatCurrency<String> = serde_json::from_str(&currency_file).unwrap();
+    let currency_bytes = currency_data
+        .codes
+        .iter()
+        .map(|y| y.as_bytes())
+        .collect::<Vec<_>>();
+    let currency_tickers = currency_bytes
+        .iter()
+        .map(|y| Ticker::try_from(*y).unwrap())
+        .collect::<Vec<_>>();
+
+    currency_tickers
```

Comment:
```suggestion
    currency_data
        .codes
        .into_iter()
        .map(|y| y.as_bytes().try_into())
        .collect()
```

Reply:
I got this error when using the updated code :

```
error[E0277]: a value of type `std::vec::Vec<polymesh_primitives::Ticker>` cannot be built from an iterator over elements of type `sc_service::Result<_, _>`
   --> src/chain_spec.rs:142:10
    |
142 |         .collect()
    |          ^^^^^^^ value of type `std::vec::Vec<polymesh_primitives::Ticker>` cannot be built from `std::iter::Iterator<Item=sc_service::Result<_, _>>`
```

Reply:
You can fix it with: `.try_into().unwrap()`